es6-promises/main.js

```diff
@@ -0,0 +1,5 @@
+import takeAChance from './take-a-chance.js';
+
+const promise = takeAChance('Tim');
+promise.then((value) => console.log(value));
+promise.catch((error) => console.log(error.message));
```

Comment:
Calling it like this will result in an unhandled promise rejection (which is why take-a-change.js line 1 is necessary).

The best (and proper) way to set up your promises is to use chaining:
`const promise = takeAChance('Tim).then(...).catch(...);`

Reply:
Fixed